### PR TITLE
feat(indexer): add video tracks info into reference

### PR DESCRIPTION
**Pull request by normanzb**
## Summary by CodeRabbit

- New Features
  - Video metadata extraction for URLs, returning optional videoTracks (codec, width, height) for videos with graceful fallbacks and improved error logging.
- Documentation
  - Updated SDK reference and OpenAPI to document videoTracks across all relevant video response types; refreshed “Defined in” links.
- Tests
  - Added comprehensive tests for video metadata extraction, covering MP4/WebM, headers, aborts, and edge cases.
- Chores
  - Added mediabunny dependency and Vite override; minor config cleanup.
  - Updated sitemap timestamps.

### CODE REVIEW

File: apps/indexer/src/utils/getImageDimension.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { IndexerAPICommentReferenceURLImageSchema } from "@ecp.eth/sdk/indexer";
 const sizeSchema = IndexerAPICommentReferenceURLImageSchema.shape.dimension;
 // It looks like commonly the image header can be retrieved within 3000 bytes, so we set a default limit of 30000 bytes
```
Comment (michalkvasnicak) [inline apps/indexer/src/utils/getImageDimension.ts:7]:
```suggestion
// It looks like commonly the image header can be retrieved within 3000 bytes, so we set a default limit
```

File: apps/indexer/src/utils/getVideoTracks.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+  if (!response.ok || !response.body) {
+    return;
+  }
+
+  if (requestAbortController.signal.aborted) {
+    return;
+  }
+
+  const contentLength = parseInt(response.headers.get("content-length") ?? "0");
```
Comment (michalkvasnicak) [inline apps/indexer/src/utils/getVideoTracks.ts:43]: Is this really always available? I mean that you can easily send a file without `content-length` should we short circuit in that case?

Comment (michalkvasnicak) [reply apps/indexer/src/utils/getVideoTracks.ts:43]: I think we can safely try to read the `MAX_VIDEO_HEADER_LENGTH` regardless of `content-length` header.

File: apps/indexer/tests/utils/getVideoTracks.test.ts [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+
+    it("should return undefined for zero content-length", async () => {
+      const videoBuffer = fs.readFileSync(
+        path.join(fixturesDir, "example-video.mp4"),
+      );
+
+      const response = new Response(videoBuffer, {
+        status: 200,
+        headers: {
+          "content-length": "0",
```
Comment (michalkvasnicak) [inline apps/indexer/tests/utils/getVideoTracks.test.ts:199]: same here

File: apps/indexer/tests/utils/getVideoTracks.test.ts [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+
+    it("should return undefined for negative content-length", async () => {
+      const videoBuffer = fs.readFileSync(
+        path.join(fixturesDir, "example-video.mp4"),
+      );
+
+      const response = new Response(videoBuffer, {
+        status: 200,
+        headers: {
+          "content-length": "-1",
```
Comment (michalkvasnicak) [inline apps/indexer/tests/utils/getVideoTracks.test.ts:218]: same here

File: package.json [hunk 5]
```diff
@@ -48,7 +48,8 @@
       "@walletconnect/universal-provider": "~2.21.1",
       "zod": "^3.25.76",
       "viem": "~2.29.2",
-      "use-sync-external-store": "^1.5.0"
+      "use-sync-external-store": "^1.5.0",
+      "vite": "6.3.5"
```
Comment (michalkvasnicak) [inline package.json:52]: why do we pin `vite` here?

File: apps/indexer/src/utils/getVideoTracks.ts (hunk 2 above)
Comment (normanzb) [reply apps/indexer/src/utils/getVideoTracks.ts]: fixed

File: apps/indexer/tests/utils/getVideoTracks.test.ts [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+
+    it("should return undefined for invalid content-length", async () => {
+      const videoBuffer = fs.readFileSync(
+        path.join(fixturesDir, "example-video.mp4"),
+      );
+
+      const response = new Response(videoBuffer, {
+        status: 200,
+        headers: {
+          "content-length": "invalid",
```
[the remark this answers is not in the document]
Comment (normanzb) [reply apps/indexer/tests/utils/getVideoTracks.test.ts]: tests updated

File: apps/indexer/tests/utils/getVideoTracks.test.ts (hunk 4 above)
Comment (normanzb) [reply apps/indexer/tests/utils/getVideoTracks.test.ts]: tests updated
